spring-cloud-gcp-samples/spring-cloud-gcp-trace-sample/src/test/java/brave/internal/HexCodec.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2013-2020 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package brave.internal;
+
+// code originally imported from zipkin.Util
+public final class HexCodec {
```

Comment:
as this is src/test ... maybe just use Long.whatever? HexCodec is mostly for performance

Reply:
I will remove this class altogether, since you've added it in `0.17.0`.